src/workers/parser/getSymbols.js

```diff
@@ -71,10 +72,27 @@ function getVariableNames(path: NodePath): SymbolDeclaration[] {
     }));
   }
 
-  return path.node.declarations.map(dec => ({
-    name: dec.id.name,
-    location: dec.loc
-  }));
+  const declarations = path.node.declarations
+    .filter(dec => dec.id.type !== "ObjectPattern")
+    .map(dec => {
+      if (dec.id.type === "ArrayPattern") {
+        return dec.id.elements.map(element => {
+          return {
+            name: element.name || element.argument.name,
+            location: element.loc
+          };
+        });
+      }
+
+      return [
```

Comment:
we probably need some more tests in fixtures :)

Reply:
I'll add some more tests